modules/Microsoft.Batch/batchAccounts/.test/encr.parameters.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
```

Comment:
Is batchAccounts one of the modules requiring a separated test for cmk?

Reply:
Honestly, by now I can't tell you anymore :/